### PR TITLE
Increase version_cmd maximum + fail hard if size exceeds

### PR DESCRIPTION
**Pull request by Jongy**
The moment `pid` becomes length of 5 or more, the string exceeds the set space:
```
In [7]: len(            "{ echo -n /proc/%d/root/; " 
   ...:             "  awk '/libphp[78]/{print $NF; exit 0} END{exit 1}' /proc/%d/maps " 
   ...:             "  || readlink /proc/%d/exe; } " 
   ...:             "| { xargs stat --printf=%%n 2>/dev/null || echo /proc/%d/exe; } " 
   ...:             "| xargs strings " 
   ...:             "| grep -Po '(?<=X-Powered-By: PHP/)\\d\\.\\d'".replace("%d", "123"))                                                                                                                  
Out[7]: 248

In [8]: len(            "{ echo -n /proc/%d/root/; " 
   ...:             "  awk '/libphp[78]/{print $NF; exit 0} END{exit 1}' /proc/%d/maps " 
   ...:             "  || readlink /proc/%d/exe; } " 
   ...:             "| { xargs stat --printf=%%n 2>/dev/null || echo /proc/%d/exe; } " 
   ...:             "| xargs strings " 
   ...:             "| grep -Po '(?<=X-Powered-By: PHP/)\\d\\.\\d'".replace("%d", "1234"))                                                                                                                 
Out[8]: 252

In [9]: len(            "{ echo -n /proc/%d/root/; " 
   ...:             "  awk '/libphp[78]/{print $NF; exit 0} END{exit 1}' /proc/%d/maps " 
   ...:             "  || readlink /proc/%d/exe; } " 
   ...:             "| { xargs stat --printf=%%n 2>/dev/null || echo /proc/%d/exe; } " 
   ...:             "| xargs strings " 
   ...:             "| grep -Po '(?<=X-Powered-By: PHP/)\\d\\.\\d'".replace("%d", "12345"))                                                                                                                
Out[9]: 256
```

I got this error `sh: 1: Syntax error: Unterminated quoted string` and needed `strace` to see what `execve`s are done, exactly. Seeing the truncated result made it clear haha.

(1024 is perhaps some exaggeration, it can probably be just 300 or we can calculate the exact number, if you'd like)

### CODE REVIEW

**Comment by adsr** [comment]
Thanks!

File: phpspy.c [hunk 1]
```diff
@@ -738,6 +738,11 @@ static void try_get_php_version(trace_target_t *target) {
             "| grep -Po '(?<=X-Powered-By: PHP/)\\d\\.\\d'",
             pid, pid, pid, pid
         );
+        if (n >= sizeof(version_cmd) - 1) {
+            log_error("version command overflow!");
+            return PHPSPY_ERR;
```
Comment (Jongy) [inline phpspy.c:743]: Oh woops. I just saw that, while I originally based this commit on some [internal version](https://github.com/Granulate/phpspy/tree/v0.6.0-g3) of ours where `try_get_php_version` has a return value, and in master it has not, so this won't build. Will send a fix promptly

Comment (Jongy) [reply phpspy.c:743]: Well it does build because we don't have `-Wall`; I did test the build but I ignored warnings :( Will fix anyway

Comment (Jongy) [reply phpspy.c:743]: Ahh good, I see you've fixed it for https://github.com/adsr/phpspy/issues/114. Thanks @adsr and sorry for the hassle :)

**Pull request merged**